WalletWasabi/Bases/PeriodicRunner.cs

```diff
@@ -104,7 +105,14 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 				var info = ExceptionTracker.Process(ex);
 				if (info.IsFirst)
 				{
-					Logger.LogError(info.Exception);
+					if (info.Exception.InnerException is TorException)
```

Comment:
We generally want to catch network errors. Can you switch off Tor in your client and see if this handles that case as well?

Reply:
Switching off Tor resulted in similar exceptions but not from Tor related codes. Catching `HttpRequestExceptions` results in fixing both clearnet and tor log when having connection errors.